README.md

```diff
@@ -1,2 +1,52 @@
-# portfolio-mobile-version
-My first mobile portfolio
+![](file:///home/moise/portfolio-mobile-version/index.html)
+
+# Portfolio: setup a Mobile Version Skeleton
+
+> This is my second project on how to design a Portfolio.
+
+![screenshot](./appScreenshot.png)
```

Comment:
- [ ] The `screenshot`  in the README file is broken. I would highly recommend fixing this link to make README professional.